prow/cmd/tackle/main.go

```diff
@@ -161,6 +175,55 @@ func selectProject(choice string) (string, error) {
 	return choice, nil
 }
 
+// selectZone returns the user-selected zone, defaulting to the current gcloud one.
+func selectZone() (string, error) {
+	const MaxZones = 20
+	var choice string
+
+	def, err := currentZone()
+	if err != nil {
+		return "", fmt.Errorf("get current zone: %v", err)
+	}
+
+	fmt.Printf("Available zones:\n")
+
+	zoneList, err := zones(MaxZones)
+	if err != nil {
+		return "", fmt.Errorf("list zones: %v", err)
+	}
+
+	for _, zone := range zoneList {
+		fmt.Println("  *", zone)
+	}
+
+	if len(zoneList) >= MaxZones {
+		fmt.Println("  ...")
+		fmt.Println("Type the name of any zone, including ones not in this truncated list")
```

Comment:
I don't think we'll ever hit this case since we limit the number of zones returned by `gcloud compute zones list` to `MaxZones`.

Reply:
@cjwagner - I think you mean this case is *always* hit since `len(zoneList)` is always equal to `MaxZones`, but I see what you mean. If might still be wise to include this check in case GCP ever reduces the number of zones or if the value of `MaxZones` is ever increased?

Reply:
🤦‍♂ Yeah that totally makes sense.

Reply:
I would expect the number of zones to change next month. I will be shocked if it is the same next year.